### PR TITLE
Document contact routing

### LINKED ISSUE #1418: Examples needed for implementing contact routing patterns
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Expected Behavior

Documentation helps operators configure their Sensu instances to route notifications to the ideal first responders.

## Current Behavior

Contact routing is possible using a combination of handlers and filters, but we need to solidify the required filters so we can provide a prescriptive guide.

## Possible Solution

Implement a filter asset (library) which makes it easy to define filters which test for the presence of a named contact in a common check or entity label.

## Context

[Contact routing as implemented in Sensu Enterprise](https://docs.sensu.io/sensu-enterprise/3.4/contact-routing/) is a very useful tool for routing notifications to the ideal first responders, but its applications are limited to Sensu Enterprise's built-in integrations.

We believe that Sensu Go currently has all the pieces required to implement contact routing through a series of handlers and filters, but probably need to provide a library to make the pattern easy to implement.

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

Adds a guide to configuring and testing contact routing using the has-contact filter asset.

## Motivation and Context

Closes #1418

### CODE REVIEW

File: content/sensu-go/5.11/guides/contact-routing.md [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+
+To complete this guide, you'll need:
+
+- a [Sensu backend][]
+- at least one [Sensu agent][]
+- [sensuctl][], [configured][] to talk to the Sensu backend
+- [curl][]
+- a [Slack webhook URL][] and three Slack channels available to receive test alerts
+
+To set up a quick testing environment, download and start the Sensu sandbox using [Vagrant and VirtualBox][] or [Docker Compose][].
```
Comment (Blue0ctober) [inline content/sensu-go/5.11/guides/contact-routing.md]: links for line 44-50 need to be added

File: content/sensu-go/5.11/guides/contact-routing.md [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+
+| filter name | expression | description
+| --- | --- | --- |
+| `contact_ops` | `has_contact(event, "ops")` | Allow events when the<br> entity or check contact<br> labels contain `ops`
+| `contact_dev` | `has_contact(event, "dev")` | Allow events when the<br> entity or check contact<br> labels contain `dev`
+| `no_contacts` | `no_contacts(event)` | Allow events without<br> entity or check contact<br> labels
+
+To add these filters to Sensu, use `sensuctl create`:
+
+{{< highlight text >}}
```
Comment (Blue0ctober) [inline content/sensu-go/5.11/guides/contact-routing.md]: Should be changed to `highlight shell`

File: content/sensu-go/5.11/guides/contact-routing.md [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+This command shows configuration examples for three handlers: `slack_ops`, `slack_dev`, and `slack_fallback`.
+Before you add these handlers to Sensu, you'll need to customize them to your Sensu and Slack instances:
+
+1. Replace the `SLACK_WEBHOOK_URL` variables with your Slack webhook.
+2. Change the channels to match channels available to receive alerts in your Slack instance.
+
+After editing the handler definitions, run the command to create the handlers.
+You can check the configuration using `sensuctl handler list --format yaml`.
+
+{{< highlight text >}}
```
Comment (Blue0ctober) [inline content/sensu-go/5.11/guides/contact-routing.md]: Should be changed to `highlight shell`

**Review by Blue0ctober — CHANGES_REQUESTED** [review]
Content looks great. Just need to add links and change out the highlighting syntax for a few of the commands. I can create a change for that on 7/22

**Review by cwjohnston — APPROVED** [review]
Looks great! 👍

**Pull request merged**